### ISSUE TITLE
Detect how long a user has been connected to a wireless access point

### ISSUE THREAD

**Issue by moyamanuel** [moyamanuel/hackquarantine2020#2]
As a user, I should be able to gain points for remaining at home. 

We can do this by detecting how long a user has been connected to the same access point.

**Comment by alyeffy** [comment]
We will need to incorporate GPS location as well into this I think. Just in case their wifi disconnects for whatever reason.

**Comment by moyamanuel** [comment]
Good point! We can track both the GPS location and access point connectivity and check the status of both if either changes. 

Thoughts or suggestions? @alyeffy

**Comment by alyeffy** [comment]
https://stackoverflow.com/questions/5888502/how-to-detect-when-wifi-connection-has-been-established-in-android/22626736

I found this thread on stackoverflow. Maybe we could allow 'grace period' of around 30 minutes if wifi gets automatically disconnected. Their streak won't be broken if they are still in the same location during the that time, and nobody new shows up in their vicinity bluetooth-wise. I wonder if it would be worth adding a 'family' feature to the app, to list devices that belong to people in your home.
